forest/src/cli/mod.rs

```diff
@@ -158,17 +160,10 @@ impl CliOpts {
                 // Parse and return the configuration file
                 read_toml(&toml)?
             }
-            None => {
-                // Check ENV VAR for config file
-                if let Ok(config_file) = std::env::var("FOREST_CONFIG_PATH") {
-                    // Read from config file
-                    let toml = read_file_to_string(&PathBuf::from(&config_file))?;
-                    // Parse and return the configuration file
-                    read_toml(&toml)?
-                } else {
-                    Config::default()
-                }
-            }
+            None => match find_default_config() {
+                Some(cfg) => cfg,
+                None => Config::default(),
+            },
```

Comment:
```suggestion
            None => find_default_config().unwrap_or_default()
```

Reply:
I think that's a good idea, will sneak it in now